sensors/01-meteorological-data.Rmd

```diff
@@ -145,14 +133,14 @@ into the command line. This uses the URL from above and the new file is named
 spectra.json.
 
 ```{sh eval=FALSE}
-curl -o spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
+curl -o spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-01&until=2017-12-31
```

Comment:
@dlebauer I thought we were changing the dates to May 2018?

Reply:
Yes - but to be consistent w/ all of season 6 (Apr 6-Aug 01) lets do the following: (starting a few days before planting on Apr 6)

```suggestion
curl -o spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-04-01&until=2017-08-01
```

Reply:
This is still 2017?